### PR TITLE
Application auto restart functionality

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

File: src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ * Utility class for managing a Java application running on the Ewon Flexy system.
+ *
+ * @since 1.2.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class SCAppManagement {
+
+  /** File path for jvmrun file */
+  private static final String JVM_RUN_PATH = "usr/jvmrun";
```
Comment (alexjhawk) [inline src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java:17]: Should be updated to `/usr/jvmrun` to follow the established convention of using `/usr`

**Review by alexjhawk — CHANGES_REQUESTED** [review]

**Review by alexjhawk — APPROVED** [review]

**Comment by TomKimsey** [comment]
@oliver-walker-hms

File: src/main/java/com/hms_networks/americas/sc/extensions/system/application/package.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<HTML>
+<BODY>
+Extension package for managing  a Java application running on the Ewon Flexy system.
```
Comment (alexjhawk) [inline src/main/java/com/hms_networks/americas/sc/extensions/system/application/package.html:3]: Just realized that there is an extra space between 'managing' and 'a'.

Comment (TomKimsey) [reply src/main/java/com/hms_networks/americas/sc/extensions/system/application/package.html]: fixed

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**